src/components/CustomTable/CustomTable.tsx

```diff
@@ -0,0 +1,143 @@
+import { Box, Table, TableBody, TableCell, TableContainer, TablePagination, TableRow } from '@mui/material';
+import { ChangeEvent, MouseEvent, useEffect } from 'react';
+import Checkbox from '@mui/material/Checkbox';
+import { CustomTableHead } from '../CustomTableHead/CustomTableHead';
+import { CustomTableProps } from './CustomTable.types';
+import { FetchError } from '../../errors/FetchError';
+
+export function CustomTable<T extends Record<PropertyKey, unknown>>(props: CustomTableProps<T>) {
+  const { toolbar, fetchUrl, selectedRowsIds, page, rows, rowsPerPage, setSelectedRowsIds, setPage, setRowsPerPage, setRows } = props;
+
+  const isSelected = (row: T) => {
+    return (
+      selectedRowsIds.filter((item) => {
+        return (row.id as string) === item;
+      }).length > 0
+    );
+  };
+
+  const pixelHeightPerRow = 53;
+  const rowsPerPageArray = [5, 10, 25];
```

Comment:
haven't we get rid of this magic-number rule ?

Reply:
fixed